### PR TITLE
Command Line Basics, Text Editors lessons: Moved Exercise section to Assignment section

### LINKED ISSUE #24236: Style Guide - Foundations/Installations: Assignments
# Foundations/Installations Section

This issue is part of the Style Guide Community Project, a way of offering members of The Odin Project community a chance to contribute to our curriculum. Users who wish to take part and contribute to any Style Guide issues should have thoroughly read the [layout style guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md) and [The Odin Project contributing guide](https://github.com/TheOdinProject/theodinproject/blob/main/CONTRIBUTING.md).
  
## Description of Issue

This issue ensures the Assignment section has the correct formatting. If this section does not exist in a lesson, it should not be added.

Each applicable lesson for this issue should meet the following bulleted requirements:

- If a lesson has a section named Exercise, Practice, or something similar, its contents is moved to the Assignment section.
    - To see a before and after example of this, checkout the previously merged pull request for https://github.com/TheOdinProject/curriculum/pull/23825, specifically the "Files changed" tab.
- The Assignment section follows the below format:
	```
	### Assignment

	<div class="lesson-content__panel" markdown="1">

	1.  A RESOURCE OR EXERCISE ITEM
	    *   AN INSTRUCTION ITEM

	</div>
	```

- Do not make any updates to any actual list numbering, indentation/spacing, etc., as this will be covered by a separate issue.
 
## Acceptance Criteria

- [ ] The following lessons are updated according to the issue description above:
	- [ ] [Text Editors](https://www.theodinproject.com/paths/foundations/courses/foundations/lessons/text-editors)
	- [ ] [Command Line Basics](https://www.theodinproject.com/paths/foundations/courses/foundations/lessons/command-line-basics-web-development-101)

### PR DESCRIPTION
**Pull request by muditchoudhary**
Complete the following REQUIRED checkboxes:

-   [x] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/theodinproject/blob/main/CONTRIBUTING.md)
-   [x] The title of this PR follows the `location of change: brief description of change` format, e.g. `Intro to HTML and CSS lesson: Fix link text`

Complete the following checkboxes ONLY IF they are applicable to your PR. You can complete them later if they are not currently applicable:
-   [x] I have previewed all lesson files included in this PR with the [Markdown preview tool](https://www.theodinproject.com/lessons/preview) to ensure the Markdown content is formatted correctly
-   [x] I have ensured all lesson files included in this PR follow the [Layout Style Guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md)

<hr>

**1. Because:**

Closes #24236

**2. This PR:**

- Removed Exercise Section from Command Line Basics Lesson.
- Add a new numbered list item to Assignment for Exercise.
- No Edit in Text Editors lesson.

**3. Additional Information:**

None.

### CODE REVIEW

File: foundations/installations/command_line_basics.md [hunk 1]
```diff
@@ -50,8 +50,18 @@ This section contains a general overview of topics that you will learn in this l
     
 2. With your newly discovered CLI super powers, practice creating a folder and a few files using the `mkdir`, `touch`, and `cd` commands introduced in the previous step. As an example, a basic website might have a main `index.html` file, a CSS stylesheet file called `style.css`, and a folder for `images`. Think about how you could create these files with the commands and put it into practice!
 
-</div>
+3. In this exercise, you will practice creating files and directories and deleting them. You'll need to enter the commands for this exercise in your terminal. If you can't recall how to open a terminal, scroll up for a reminder.
```
Comment (thatblindgeye) [inline foundations/installations/command_line_basics.md:53]: For this line, I think we can reword it a little bit:

```suggestion
3. Let's practice creating files and directories and deleting them! You'll need to enter the commands for the steps below in your terminal. If you can't recall how to open a terminal, scroll up for a reminder.
```

Or something along those lines.

**Review by thatblindgeye — CHANGES_REQUESTED** [review]
Thanks for making this update! I made a small change request below. Additionally, would you mind making an update to the "text editors" lesson and just updating the "Additional resources" heading to "Additional Resources"?

**Comment by muditchoudhary** [comment]
Thank You for telling me the suggested changes. I worked on it as stated.
Other than that I reword the end message in practice. Please see if it's good.

**Review by thatblindgeye — COMMENTED** [review]
Looks great! There's just a merge conflict that needs to be resolved first. Do you want to try resolving it?

**Comment by muditchoudhary** [comment]
Yes, sure!!

**Review by thatblindgeye — APPROVED** [review]
Thank you for working on this!

**Pull request merged**